### PR TITLE
[Statistics] Test Plan & README Update to test widgets

### PR DESCRIPTION
**Pull request by sruthymathew123**
- [x] Have you updated related documentation?

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)
#6441

### CODE REVIEW

File: modules/statistics/test/TestPlan.md [hunk 1]
```diff
@@ -30,6 +30,14 @@
 13. Returning to the behavioural statistics page, verify that if you do not have the `access_all_profiles` permission you should not be able to click through to forms for candidates from other sites.
 14. Check the breadcrumbs from Completion stats page back to BVL stats tab within Statistics appear and allow you to use the `Back` button to previous pages (future feature).
 
+### Dashboard Widget
+1. Go to the LORIS dashboard. 
+     - Ensure that the "My Tasks" on the LORIS dashboard has a "Incomplete Forms"
```
Comment (driusan) [inline modules/statistics/test/TestPlan.md:35]: Isn't that for the conflict resolver, not the statistics module?

The statistics module provides the study progression charts..

Comment (sruthymathew123) [reply modules/statistics/test/TestPlan.md:35]: @driusan I don't know if I misunderstand something. Conflict Resolver is associated with Data Conflicts item, right? and Incomplete Forms to Behavioural statistics.

Comment (driusan) [reply modules/statistics/test/TestPlan.md]: sorry, Incomplete Forms comes from the instruments module.. still not statistics.

**Comment by ridz1208** [comment]
@sruthymathew123 @nicolasbrossard 

this PR seems to overlap with #6463 can you decide which one should go and close the other one please

**Comment by sruthymathew123** [comment]
Keeping Nic's PR #6463  and closing mine

**Pull request closed without merging**